launch/test/launch/frontend/test_parser.py

```diff
@@ -39,11 +40,12 @@ def test_invalid_launch_extension():
 
         assert mock_ep.called
         assert mock_warn.call_args
-        assert mock_warn.call_args.args[0].startswith('Failed to load')
+        assert mock_warn.call_args[0][0].startswith('Failed to load')
```

Comment:
> Changed in version 3.8: Added args and kwargs properties.

Just dropping a link in case someone looks at this PR in the future: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_args